packages/collection-model/lib/extended-model.js

```diff
@@ -1,89 +1,108 @@
-var MongoDBCollection = require('./collection-model');
```

Comment:
Should we just rename `extended-model` to `model` at this point?

Reply:
Totes, I was kinda hoping the diff wouldn't be so useless 😆

Reply:
Done in 67891d